cpp/rrtmgp/mo_gas_optics_rrtmgp.h

```diff
@@ -2050,28 +2060,40 @@ class GasOpticsRRTMGPK : public OpticalPropsK {
     Kokkos::parallel_for( MDRangeP<2>({0,0}, {ngpt,ncol}) , KOKKOS_LAMBDA (int igpt, int icol) {
       toa_src(icol,igpt) = this_solar_src(igpt);
     });
+
+    pool::dealloc(jtemp.data(), jtemp.size());
```

Comment:
Minor, but have you consider adding an overload that takes the view and queries for ptr and size internally? Might be a tad more legible.

Reply:
Good idea.